src/model/fruit.py

```diff
@@ -1,11 +1,9 @@
+from dataclasses import dataclass
 from typing import Tuple
 
 
+@dataclass
 class Fruit:
```

Comment:
Mmmmm I see, this is a dataclass. Then maybe it would be good to have a helper method that generates an instance of this class in a random position from the window dimensions